src/Main.java

```diff
@@ -0,0 +1,79 @@
+public class Main {
+
+
+    public static void main(String[] args) {
+
+
+        Players nicola = new Players();
+        Players philipp = new Players();
+        Game match = new Game(nicola, philipp);
+        Message nicolaScores = new Message(nicola, 1);
+        Message philippScores = new Message(philipp, 1);
+        Message nicolaReset = new Message(nicola, 3);
+        Message philippReset = new Message(philipp,3);
+
+       // match.playerTwo = new Players()
+
+        match.buttonWasPressedWithValue(nicolaScores);
```

Comment:
Wouldn't be better to create a new message each time you execute this function?

`match.buttonWasPressedWithValue(new Message(...))`

Reply:
That's included in the ___new version___, i think